src/components/ContextMenuItem.tsx

```diff
@@ -38,9 +38,13 @@ type ContextMenuItemProps = {
     isFocused?: boolean;
 };
 
+type ContextMenuItemRef = {
```

Comment:
```suggestion
type ContextMenuItemHandle = {
```

nit, just because we have been using `...Handle` in other components.